content_api/publish/service.py

```diff
@@ -33,7 +33,7 @@ def create(self, docs, **kwargs):
         return ids
 
     def _create_doc(self, doc, **kwargs):
-        _id = doc[config.ID_FIELD]
+        _id = doc['guid']
```

Comment:
in `def create`,  `'guid'` is deleted from doc. It's not an issue ?
Also, before the key was coming from the `config`. Shouldn't we keep this key configurable ?